src/boot/boot.c

```diff
@@ -326,15 +327,16 @@ static void print_status(Config *config, char16_t *loaded_image_path) {
                 printf("    default (one-shot): %ls\n", config->entry_oneshot);
         if (config->entry_saved)
                 printf("           saved entry: %ls\n", config->entry_saved);
-        printf("                editor: %ls\n", yes_no(config->editor));
-        printf("          auto-entries: %ls\n", yes_no(config->auto_entries));
-        printf("         auto-firmware: %ls\n", yes_no(config->auto_firmware));
-        printf("         auto-poweroff: %ls\n", yes_no(config->auto_poweroff));
-        printf("           auto-reboot: %ls\n", yes_no(config->auto_reboot));
-        printf("                  beep: %ls\n", yes_no(config->beep));
-        printf("  reboot-for-bitlocker: %ls\n", yes_no(config->reboot_for_bitlocker));
-        printf("       reboot-on-error: %s\n",  reboot_on_error_to_string(config->reboot_on_error));
-        printf("    secure-boot-enroll: %s\n",  secure_boot_enroll_to_string(config->secure_boot_enroll));
+        printf("                  editor: %ls\n", yes_no(config->editor));
+        printf("            auto-entries: %ls\n", yes_no(config->auto_entries));
+        printf("           auto-firmware: %ls\n", yes_no(config->auto_firmware));
+        printf("           auto-poweroff: %ls\n", yes_no(config->auto_poweroff));
+        printf("             auto-reboot: %ls\n", yes_no(config->auto_reboot));
+        printf("                    beep: %ls\n", yes_no(config->beep));
+        printf("    reboot-for-bitlocker: %ls\n", yes_no(config->reboot_for_bitlocker));
+        printf("          reboot-on-error: %s\n",  reboot_on_error_to_string(config->reboot_on_error));
+        printf("       secure-boot-enroll: %s\n",  secure_boot_enroll_to_string(config->secure_boot_enroll));
+        printf("secure-boot-enroll-action: %s\n",  secure_boot_enroll_action_to_string(config->secure_boot_enroll_action));
```

Comment:
hmm, the printf() lines in the if block above would also need to be reindented. maybe just abbreviate this field here though, might be easier.